libqtile/backend/x11/xcursors.py

```diff
@@ -105,7 +105,7 @@ def finalize(self):
     def _setup_xcursor_binding(self):
         try:
             xcursor = ffi.dlopen("libxcb-cursor.so.0")
-        except OSError:
+        except Exception:
```

Comment:
Can you log the contents of the exception to see what it actually is?

Reply:
would we see the log in the CI tests logs? IIRC pytest catches everything, but maybe not warnings yeah